Sources/OutlineView/OutlineViewDragAndDrop.swift

```diff
@@ -0,0 +1,139 @@
+import Foundation
+import AppKit
+
+/// A protocol for use with `OutlineView`, implemented by an object (most likely
+/// the data source) to handle receiving items that are dragged into the `OutlineView`.
+@available(macOS 10.15, *)
+public protocol DropReceiver {
+    associatedtype DataElement: Identifiable
+    
+    /// Given a dragged `NSPasteboardItem`, attempts to read the data in the item
+    /// into a usable object for the `OutlineView`'s data source.
```

Comment:
Again, I would refrain for using "object". In Swift "object" specifically refers to reference types like classes and actors, which might not be the case here.

How about:
> Converts the `NSPasteboardItem` received by a drag-and-drop operation to the data element of an OutlineView.

Reply:
Good call. Fixed and got rid of other "object" references (still showing some of my old ObjC coding habits). https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3